pkg/ingestor/parser/clearlydefined/clearlydefined_test.go

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright 2022 The GUAC Authors.
```

Comment:
do we care about copyright years? (I see some here are 2024, some are 2022)

Reply:
ah, I thought I caught this...can be fixed in another PR.